packages/core/http/core-http-server-internal/src/http_server.ts

```diff
@@ -516,6 +516,7 @@ export class HttpServer {
   }
 
   private configureRoute(route: RouterRoute) {
+    const optionsLogger = this.logger.get('http', 'server', this.name, 'options');
     this.log.debug(`registering route handler for [${route.path}]`);
```

Comment:
I wonder if extending this log entry with the values suggested in https://github.com/elastic/kibana/pull/152621/files/bf7a3c36cb4394b06a3b3fa09a0553c3f958bfed#r1126120651 would suffice. Both entries look too similar and are logged at the same time. If the only reason is so that we can use a new logger, I wonder if we can use it here (as long as it's nested, it should still log with the same config)

Reply:
I thought about that but went a new logger route to be able to silence the extra info or to later remove the logger completely without losing the original log